packages/create-redwood-app/src/create-redwood-app.js

```diff
@@ -169,12 +169,12 @@ const installNodeModulesTasks = ({ newAppDir }) => {
 const style = {
   error: chalk.bold.red,
   warning: chalk.keyword('orange'),
-  success: chalk.hex('#c3e88d'),
+  success: chalk.greenBright,
   info: chalk.grey,
   header: chalk.bold.underline.hex('##e8e8e8'),
   cmd: chalk.hex('#808080'),
   redwood: chalk.hex('#ff845e'),
-  love: chalk.reset.hex('#de1d10'),
+  love: chalk.redBright,
 }
 
```

Comment:
This will look as follows

![image](https://user-images.githubusercontent.com/865493/93993714-5cf60880-fd8f-11ea-92ab-411eb15eeda7.png)

Reply:
This looks really good <3, learning a ton here :)